src/components/custom/eui/EUIIntegration.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useRef } from "react";
+import Script from "next/script";
+
+const EUIIntegration = ({ height }) => {
+    const euiRef = useRef(null);
+
+    useEffect(() => {
+        const eui = euiRef.current;
+        eui.dataSources = [
+            "https://apps.humanatlas.io/hra-api/v1/sennet/rui_locations.jsonld",
+        ];
+    }, []);
+
+    return (
+        <>
+            <ccf-eui
+                style={{ height: height }}
+                ref={euiRef}
+                theme="sennet"
+                header="false"
+                use-remote-api="false"
+                hubmap-data-url=""
```

Comment:
For private data you can either pass the token in here (add a `SNT-` prefix to the token) as an atribute or in code above around line 9.
```
hubmap-token="SNT-{{ token}}"
```
(apologies that it still says hubmap- and not something more generic)